### PR TITLE
Frontend alt design

### LINKED ISSUE #14: Can remove a link
**Frontend**
- Pressing a link, a delete option will appear allowing a user to remove the link.
- Pressing a link will highlight the link which can be then removed by pressing delete button.

**Backend**
- DELETE-request will be sent to "/api/link/<link_id>" which will remove the link from database.

### LINKED ISSUE #29: Link prerequisites to courses
Toggle concept for current course to activate prerequisite selection. Toggle concepts in the courses shown as prerequisites for the current course to make the concepts prerequisites for the current concept active for selection.

### LINKED ISSUE #31: Column with courses that can be added as prerequisites for the current course.

### LINKED ISSUE #32: Refactor view showing all courses. Click a course to move to view curriculum page for it.

### LINKED ISSUE #33: Column with current course and its concepts.

### LINKED ISSUE #34: Container containing CourseColumns containing its concepts.

### PR DESCRIPTION
**Pull request by stadibo**
- Container center of screen containing CourseColumns, containing its concepts. Resolves #34 
- Column showing the active course and its concepts. Can add new concept. Resolves #33 
- Tray (column) with all courses which can be added to active course as prerequisites. Resolves #31 
- Refactored frontend: course menu, curriculum mapper. Resolves #32 Resolves #29 Resolves #14

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -34,4 +34,244 @@ body {
...
+  flex-direction: column;
+  flex-basis: 100%;
+}
+
+.course-view {
+  display: flex;
+  overflow: hidden; 
+}
+
+.curri-button {
```
Comment (stadibo) [inline frontend/src/App.css]: We need to change the style names to fit the application better. Also clear out unnecessary stylings.

File: frontend/src/components/course/ActiveCourse.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              key={concept.id}
+              activeConceptId={activeConceptId}
+              activateConcept={activateConcept}
+            />
+          )}
+        </div>
+
+      </div>
+      <HeaderButton onClick={openModal} text='+' />
+      <Modal
```
Comment (stadibo) [inline frontend/src/components/course/ActiveCourse.js]: The Modal component could be wrapped as a utility component, however, might make it to inflexible.

File: frontend/src/components/course/CourseView.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    refetchQueries: [{
+      query: COURSE_PREREQUISITE_COURSES,
+      variables: { id: course_id }
+    }]
+  })
+
+  const createConcept = useMutation(CREATE_CONCEPT, {
+    refetchQueries: [{
+      query: ALL_COURSES
+    }]
```
Comment (stadibo) [inline frontend/src/components/course/CourseView.js]: FETCH_COURSE was creating unexpected behavior, and is not refetched after creating concept for course. Need to investigate further.

File: frontend/src/components/course/CourseResults.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
```
Comment (d471061c) [inline frontend/src/components/course/CourseResults.js]: This file doesn't seem to be used, could be removed.

**Pull request merged**